tests/integration/test_decompositions_integration.py

```diff
@@ -554,7 +554,7 @@ def test_Pgate_decomp_equal(self, setup_eng, s, tol):
         eng.run(prog)
         assert np.all(eng.backend.is_vacuum(tol))
 
-    @pytest.mark.parametrize('s', np.linspace(0., 0.6, 5))
+    @pytest.mark.parametrize('s', np.linspace(-0.5, 0.6, 5))  # FIXME include 0 in testset
```

Comment:
@smite What needs to be fixed here?

Reply:
For now CXgate causes a divide-by-zero error when `s=0`. When Sympy 1.5 is released we can fix it, and then this test should include the value zero in the testset.

Reply:
Can you add this information to the FIXME comment

Reply:
Done.